tsconfig.json

```
@@ -25,6 +25,6 @@
     "src"
   ],
   "exclude": [
-    "node_modules/@elastic/eui/src"
+    "node_modules/@elastic/eui/src",
```

nit: this might break a regular `JSON.parse` - if not needed, let's keep the file compatible
```suggestion
    "node_modules/@elastic/eui/src"
```

---

Pushed cb46b97d027b3f420613b3bedafbe265b5adda9f to revert to `main`